docs/framework/app-domains/build-multifile-assembly.md

```diff
@@ -81,15 +81,15 @@ This article explains how to create a multifile assembly and provides code that
 
 2. Use the following command to compile this code:
 
-   ```cpp
+   ```console
```

Comment:
If we change these to console, all three will show up for any selected language and confuse costumers.

Reply:
yeah, to change that to console, we'd have to use tabs. not sure if it's worth it in these topics.
There's no support for double identifiers.